bridges/evm/contracts/StoreProofLib.sol

```diff
@@ -25,6 +34,9 @@ library StoreProofLib {
     ));
   }
 
+  /**
+   * @dev Returns a computed store hash
```

Comment:
How is this related to Tendermint store structure? Why do we need to do `sha256` multiple times? Please explain it here so people that come read this later understand what's going on.